File: MyHelper.ClientApp/src/app/shared/models/friend/friend-view.model.ts
import { AppUserViewModel } from '../user/app-user-view.model';
import { FriendRequestFlag, FriendRequestDirection } from '../../utilities/enums';

export class FriendViewModel
  extends AppUserViewModel {
    public friendRequestDirection: FriendRequestDirection;
    public friendRequestFlag: FriendRequestFlag;
}

File: MyHelper.Api/Models/Friend/FriendViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using MyHelper.Api.Core;
using MyHelper.Api.Models.User;

namespace MyHelper.Api.Models.Friend
{
    public class FriendViewModel : AppUserViewModel
    {
        public EFriendRequestDirection FriendRequestDirection { get; set; }

        public EFriendRequestFlag FriendRequestFlag { get; set; }
    }
}
